### DISCUSSION TITLE
MVVM Toolkit - Trigger "AlsoNotifyChangeFor"

### DISCUSSION THREAD

**Discussion by Phil4121** [CommunityToolkit/dotnet#165] [General]
Hello!

In my sample Project I've got a model. The model looks like this:

```
public partial class BaseModel : ObservableObject, IBaseModel
{
    [ObservableProperty]
    private string id;

    [ObservableProperty]
    private string name;

    [ObservableProperty]
    private bool isActive;

    [ObservableProperty]
    private bool isNew;

    public BaseModel(bool isNew)
    {
        IsNew = isNew;
        IsActive = true;
    }
}
```

In my ViewModel I'm using this model like this:

```
        [AlsoNotifyChangeFor(nameof(CanDeleteSample))]
        [AlsoNotifyChangeFor(nameof(CanSaveSample))]
        [AlsoNotifyCanExecuteFor(nameof(SaveSampleCommand))]
        [AlsoNotifyCanExecuteFor(nameof(DeleteSampleCommand))]
        [ObservableProperty]
        private BaseModel sample = new();
```

If I first set "sample" with this "new()" initialization CanDeleteSample, CanSaveSample,... are notified and everything works fine.
But if I change the value of, for example, "Name" no notification is done.

Is there a way to handle the notification if a property of "sample" has been changed? Further is there also a way to execute the notifications if I use a wrapped object withing the BaseModel class?

Thx

**Comment by michael-hawker** [comment]
FYI @Sergio0694, moved this over here from the WCT discussions.
